packages/agoric-cli/src/commands/ec.js

```diff
@@ -6,8 +6,6 @@ import { Command } from 'commander';
 import { makeRpcUtils, storageHelper } from '../lib/rpc.js';
 import { outputExecuteOfferAction } from '../lib/wallet.js';
 
-const { vstorage, fromBoard, agoricNames } = await makeRpcUtils({ fetch });
```

Comment:
ah yes. I think this was made one in `rpc.js` before but that was bad so it was factored it into a maker, but then didn't go the distance to only make where used.